assets/js/components/notifications/index.js

```diff
@@ -51,6 +53,12 @@ if ( setup.needReauthenticate ) {
 		addAuthNotification, 1 );
 }
 
+if ( featureFlags.userInput.enabled ) {
+	addFilter( 'googlesitekit.ErrorNotification',
+		'googlesitekit.UserInputSettings',
+		addUserInputSettings, 1 );
```

Comment:
This shouldn't use the `googlesitekit.ErrorNotification` filter since it's not an error. It makes it show up in more locations than it should. Per the ACs, the notification should only appear on the dashboard, e.g. it should use `googlesitekit.DashboardNotifications`.